py/torch_tensorrt/dynamo/utils.py

```diff
@@ -711,7 +711,13 @@ def get_output_metadata(
 
 def get_output_dtypes(output: Any, truncate_doulbe: bool = False) -> List[dtype]:
     output_dtypes = []
-    if isinstance(output, torch.fx.node.Node):
+    if isinstance(output, int):
```

Comment:
you can just use type conversion here instead of a switch statement. 

```py
output_dtypes.append(dtype._from(output))
```

If dtype doesnt support python types we should probably add it